packages/block-library/src/reset.scss

```diff
@@ -8,19 +8,18 @@
 // We use :where to keep specificity minimal.
 // https://css-tricks.com/almanac/selectors/w/where/
 :where(.editor-styles-wrapper) {
-	/**
-	* The following styles revert to the browser defaults overriding the WPAdmin styles.
-	* This is only needed while the block editor is not being loaded in an iframe.
-	*/
-	font-family: serif; // unfortunately initial doesn't work for font-family.
-	font-size: initial;
-	line-height: initial;
-	color: initial;
+	// The following styles revert to the browser defaults overriding the WPAdmin styles.
+	html & {
+		font-family: serif; // unfortunately initial doesn't work for font-family.
+		font-size: initial;
+		line-height: initial;
+		color: initial;
 
-	// Many themes with white backgrounds load editor styles but fail to also provide
-	// an explicit white background color, assuming a white editing canvas.
-	// So to match browser defaults, we provide a white default here as well.
-	background: #fff;
+		// Many themes with white backgrounds load editor styles but fail to also provide
+		// an explicit white background color, assuming a white editing canvas.
+		// So to match browser defaults, we provide a white default here as well.
+		background: #fff;
+	}
```

Comment:
The admin styles seem to specifically target the `body`. Could we move these particular styles out of the `here(.editor-styles-wrapper)` block and under `body`, maybe with a link to which styles are being reset?

Also when did this break? Where you able to pinpoint the specific commit or release?

Reply:
Cc @aaronrobertshaw in case it rings a bell

Reply:
The selector needs at least `0-0-1` specificity to override `body`, which it had before [removing `html` in 46752](https://github.com/WordPress/gutenberg/pull/46752/files#diff-bda8e417e9cfc8724167f4c82aab1d0d205245c51360a8afa4bba99ba078e6dbR10).

I made #61959 to restore that selector exactly as it was for the whole set, and Aaron already suggested to [isolate the change](https://github.com/WordPress/gutenberg/pull/61959#discussion_r1618026389) to only the first selector of the stylesheet.

Reply:
In that case, we should probably change the selector to:

```css
div:where(.editor-styles-wrapper),
body:where(.editor-styles-wrapper)
```

So that it overrides the admin styles for both an iframed and non-iframe editor.

Reply:
`html :where(.editor-styles-wrapper)` applied to either element in WordPress 6.3, and I just made a quick edit _directly_ in `wp-includes\css\dist\block-library\reset.css` with 6.6 beta to confirm that it still covers both elements.

iframe:

![iframe editor with corrected background and font size](https://github.com/WordPress/gutenberg/assets/17100257/284d0975-53be-43f6-bdb4-aff0bd7d528e)

non-iframe:

![non-framed editor with element inspector open](https://github.com/WordPress/gutenberg/assets/17100257/5e41fc51-8dbc-4866-9961-ceb897b40d38)

(the theme for these screenshots is Twenty Thirteen)

Reply:
Why not just restore it to https://github.com/WordPress/gutenberg/pull/46752/files#diff-bda8e417e9cfc8724167f4c82aab1d0d205245c51360a8afa4bba99ba078e6dbR10?

Reply:
Restoring is what I tried first :) #61959

Reply:
Yes, so we only want it to affect the top level styles. I guess I meant, why not take out that section and do it in a separate block? I find that a bit more readable than `html &`.